docs/components/base/wheeled.md

```diff
@@ -15,7 +15,8 @@ component_description: "Supports mobile wheeled robotic bases with motors on bot
 A `wheeled` base supports mobile robotic bases with motors on both sides (differential steering).
 To configure a `wheeled` base as a component of your machine, first configure the [board](/components/board/) controlling the base and any [motors](/components/motor/) attached to the base.
 
-Configure a `wheeled` base as follows:
+Make sure to first physically assemble the base, connect it to your machine's computer, and turn it on if you want to be able to test the base as you configure it.
```

Comment:
Sorry - this is a bit annoying but if we frontload the reason then they can skim past it as they're reading if it's not relevant. (Also shortens the sentence)
```suggestion
If you want to test your base as you configure it, physically assemble the base, connect it to your machine's computer, and turn it on.
```